posix.mak

```diff
@@ -8,6 +8,12 @@
 #
 # make -f posix.mak rsync
 #
+#
+# This makefile supports 3 different builds latest, prerelease, and release. The
+# first 2 are live on the website and build from the latest released version or
+# master (prerelease). The release build is used for the documentation that is
+# shipped with binary releases. For the latter the LATEST version is not yet
+# published at build time, hence a few things differ from a prerelease build.
```

Comment:
How about mentioning how to activate each build? Might be worth mentioning that latest/prerelease are not actually different targets but are just built separately from the default target, and release is activated by setting an option.